lib/policy_machine/version.rb

```diff
@@ -1,3 +1,3 @@
 class PolicyMachine
-  VERSION = "2.1.3"
+  VERSION = "3.1.0"
```

Comment:
hadn't kept that up to date in a while i see lol

Reply:
so 3.1.0 sounds correct to you? last version bump (not in this file, but in general) was 3.0.1 => 3.0.2 for the addition of the `pluck` method.

Reply:
We probably should have bumped `3.1.0` last time. I'm fine doing it this time.